storage/aptosdb/src/db_debugger/checkpoint/mod.rs

```diff
@@ -0,0 +1,37 @@
+// Copyright (c) Aptos
+// SPDX-License-Identifier: Apache-2.0
+
+use crate::{db_debugger::common::DbDir, LEDGER_DB_NAME, STATE_MERKLE_DB_NAME};
+use anyhow::{ensure, Result};
+use clap::Parser;
+use std::{fs, path::PathBuf};
+
+#[derive(Parser)]
+#[clap(about = "Make a DB checkpoint by hardlinks.")]
+pub struct Cmd {
+    #[clap(flatten)]
+    db_dir: DbDir,
+
+    #[clap(long, parse(from_os_str))]
+    output_dir: PathBuf,
+}
+
+impl Cmd {
+    pub fn run(self) -> Result<()> {
+        ensure!(!self.output_dir.exists(), "Output dir already exists.");
+        fs::create_dir_all(&self.output_dir)?;
```

Comment:
Share the code with https://github.com/aptos-labs/aptos-core/blob/main/execution/executor-benchmark/src/lib.rs#L55 to make our life easier when adding more DBs / sharding DB?

Reply:
done